flathub.json

```
@@ -0,0 +1,5 @@
+{
+  "skip-arches": [
+    "arm"
```

```suggestion
    "aarch64"
```
arm was building just fine :)